src/main/java/org/spongepowered/api/item/inventory/InventoryProperties.java

```diff
@@ -45,50 +45,47 @@
+    /**
+     * Represents the plugin that created this inventory.
+     */
+    public static final Property<PluginContainer> PLUGIN = DummyObjectProvider.createFor(Property.class, "PLUGIN");
 
-    public static final Property<Integer> MAX_STACK_SIZE = null;
+    /**
+     * Represents the maximum stack size of slots in an inventory.
+     */
+    public static final Property<Integer> MAX_STACK_SIZE = DummyObjectProvider.createFor(Property.class, "MAX_STACK_SIZE");
 
     /**
      * Represents the {@link EquipmentType} that the target inventory supports.
      */
-    public static final Property<EquipmentType> EQUIPMENT_TYPE =
-            DummyObjectProvider.createFor(Property.class, "EQUIPMENT_TYPE");
+    public static final Property<EquipmentType> EQUIPMENT_TYPE = DummyObjectProvider.createFor(Property.class, "EQUIPMENT_TYPE");
 
     /**
      * Represents the index of a slot.
      */
-    public static final Property<Integer> SLOT_INDEX =
-            DummyObjectProvider.createFor(Property.class, "SLOT_INDEX");
+    public static final Property<Integer> SLOT_INDEX = DummyObjectProvider.createFor(Property.class, "SLOT_INDEX");
 
     /**
      * A property which represents the position of a slot within a grid. Bear in mind
      * that this property should be retrieved from the relevant parent, since a
      * slot may have multiple parent inventories.
      */
-    public static final Property<Vector2i> SLOT_POSITION =
-            DummyObjectProvider.createFor(Property.class, "SLOT_POSITION");
+    public static final Property<Vector2i> SLOT_POSITION = DummyObjectProvider.createFor(Property.class, "SLOT_POSITION");
 
     /**
      * A property which represents a "side" for a particular slot, for use in querying "sided inventories".
      */
-    public static final Property<Direction> SLOT_SIDE =
-            DummyObjectProvider.createFor(Property.class, "SLOT_SIDE");
+    public static final Property<Direction> SLOT_SIDE = DummyObjectProvider.createFor(Property.class, "SLOT_SIDE");
 
     /**
      * Represents the {@link UUID} of a inventory.
      */
-    // TODO not present in impl for query atm.
-    public static final Property<UUID> UNIQUE_ID =
-            DummyObjectProvider.createFor(Property.class, "UNIQUE_ID");
+    public static final Property<UUID> UNIQUE_ID = DummyObjectProvider.createFor(Property.class, "UNIQUE_ID");
 
     /**
-     * Represents the title of a inventory, is viewable by players looking at the inventory.
+     * Represents the title of a inventory, is viewable by players looking at the inventory. Not all inventories have a title.
      */
-    // TODO maybe not present in impl? most inventories dont have a title. also title only shows in Containers
-    public static final Property<Text> TITLE =
-            DummyObjectProvider.createFor(Property.class, "TITLE");
+    public static final Property<Text> TITLE = DummyObjectProvider.createFor(Property.class, "TITLE");
 
```

Comment:
Originally for the capacity and dimension property, my thought that it would be useful to find subinventories in built inventories. But maybe it would be better to "tag" inventories instead and use that tag to query for them. Or this could be the unique id, if it's used uniquely per inventory (and children) and not globally.